e2e_tests/tests/all_providers/mod.rs

```diff
@@ -5,6 +5,8 @@ use parsec_client::core::interface::operations::list_providers::Uuid;
 use parsec_client::core::interface::requests::{AuthType, Opcode, ProviderID, Result};
 use std::collections::HashSet;
 
+mod cross;
```

Comment:
Is that used?

Reply:
Oh this might be needed so that test are executed? Even without `pub` 🤯

Reply:
Indeed, it's only about the tests being in _some_ module